index.css

```
@@ -0,0 +1,390 @@
+body
+{
+    font: 18px Helvetica, serif;
+    color: #444;
+    width: 99%;
+}
+
+header
+{
+    font: 32px 'PT SansNarrow';
+    text-align: center;
+    line-height: .75rem;
+    position: fixed;
+    width: 99%; /* нужно отдельно указать, равный body - из-за position */
+    z-index: 2000;
+    border: 2px solid #aaa;
+    background-color: #fff;
+    margin-top: -10px;
+}
+
+header > p::first-letter
+{
+    color: #f00;
+}
+
+footer,
+main
+{
+    border: 2px solid #aaa;
+    border-top: 0;
+    width: 100%; /* не 99, а 100, ведь наследует 99 от body */
+    overflow: hidden;
+}
+
+/* main */
+
+.carousel
+{
+    display: inline-block;
+    text-align: center;
+    padding: 100px 0 20px;
+    margin: 0 auto;
+    white-space: nowrap;
+}
+
+input
+{
+    display: none;
+}
+
+.cat-box
+{
+    width: 25vw;
+    vertical-align: middle;
+    padding: 2vw;
+    display: inline-block;
+}
+
+.cat-photo
+{
+    width: 25vw;
+    height: 25vw;
+    object-fit: cover;
+}
+
+/* боковые котики меньше и прозрачнее */
+
+.inp-cat1:checked:checked ~ .cat1 > label > .cat-photo,
+.inp-cat2:checked:checked ~ .cat2 > label > .cat-photo,
+.inp-cat3:checked:checked ~ .cat3 > label > .cat-photo,
+.inp-cat4:checked:checked ~ .cat4 > label > .cat-photo,
+.inp-cat5:checked:checked ~ .cat5 > label > .cat-photo,
+.inp-cat6:checked:checked ~ .cat6 > label > .cat-photo,
+.inp-cat7:checked:checked ~ .cat7 > label > .cat-photo,
+.inp-cat8:checked:checked ~ .cat8 > label > .cat-photo,
+.inp-cat9:checked:checked ~ .cat9 > label > .cat-photo,
+.inp-cat10:checked:checked ~ .cat10 > label > .cat-photo
```

Зачем такая вложенность?

```suggestion
.inp-cat10:checked:checked ~ .cat10 .cat-photo
```